packages/webpack/src/runtime/runtimeUnlocked.js

```diff
@@ -0,0 +1,47 @@
+/// <reference path="./lavamoat.d.ts" />
+/* global LAVAMOAT */
+
+const { create, freeze, defineProperty } = Object
+const warn = typeof console === 'object' ? console.warn : () => {}
+warn('LavaMoat: using unlocked runtime')
+
+const { NAME_globalThis, NAME_scopeTerminator, NAME_runtimeHandler } =
+  LAVAMOAT.ENUM
+
+/**
+ * Wraps the webpack runtime with Lavamoat security features.
+ *
+ * @param {string} resourceId - The identifier of the resource.
+ * @param {any} runtimeKit - The runtime kit containing bits from the webpack
+ *   runtime.
+ * @returns {Object} - An object containing the wrapped runtime and other
+ *   related properties.
+ */
+const lavamoatRuntimeWrapper = (resourceId, runtimeKit) => {
+  let { module } = runtimeKit
+  const runtimeHandler = runtimeKit
```

Comment:
why the assignment?

Reply:
um. Not sure, might be an indication that I initially intended to make a copy

Reply:
Actually, I think you spotted a bug.

Reply:
fixed by making a shallow copy